src/main/java/net/snowflake/client/jdbc/SnowflakeDatabaseMetaData.java

```diff
@@ -137,6 +121,14 @@ private void raiseSQLExceptionIfConnectionIsClosed() throws SQLException {
     }
   }
 
+  private void sendInBandTelemetryMetadataMetrics(String functionName, String showCommand) {
+    ObjectNode ibValue = mapper.createObjectNode();
+    ibValue.put("type", TelemetryField.METADATA_METRICS.toString());
+    ibValue.put("function", functionName);
+    ibValue.put("show command", showCommand);
```

Comment:
please don't include space in the key. It is often used in the query, e.g., message:query_id.

Reply:
Okay, makes sense! I removed the spaces and added in the function parameters and query ID to the message as well.